telethon/types/_custom/message.py

```diff
@@ -1264,6 +1264,17 @@ async def unpin(self):
             return await self._client.unpin_message(
                 await self.get_input_chat(), self.id)
 
+    async def react(self, reaction=None):
+        # TODO Constantly checking if client is a bit annoying,
+        #      maybe just make it illegal to call messages from raw API?
+        #      That or figure out a way to always set it directly.
```

Comment:
This comment no longer really applies, since raw API messages are not this custom message anymore. But this class needs a general cleanup overall regardless.

Reply:
i removed that comment 😅